tests/integration/api/istio_test.go

```diff
@@ -215,9 +215,10 @@ var _ = Describe("Istio resource", Ordered, func() {
 	Describe("update", func() {
 		var workloadNs *corev1.Namespace
 		rev := &v1.IstioRevision{}
+		baseVersion, newVersion := istioversion.GetBaseAndNewVersion()
 
 		BeforeAll(func() {
-			if istioversion.Old == "" {
+			if len(istioversion.List) < 2 {
```

Comment:
In theory this can be false, while baseVersion and newVersion are not set.
```suggestion
			if baseVersion == "" || newVersion == "" {
```